pkg/rules/engine/engine.go

```diff
@@ -96,6 +96,7 @@ func NewEngine(sigs []detect.Signature, sources EventSources, output chan detect
 			engine.logger.Printf("error initializing signature %s: %v", meta.Name, err)
 			continue
 		}
+		engine.stats.Signatures.Increment()
```

Comment:
Nice catch! Can we have a unit test so we won't delete this line accidentally in the future?

Reply:
Thanks, but I caught myself because I'm the one who added these stats and forgot this ;)

I was actually thinking maybe we should use LoadSignature here instead, I'm not sure why there is this duplicate logic (do you know why). WDYT? We could still unit test it obviously.

Reply:
I uploaded a refactor which includes a test for signature loading, which also tests this increment, check it out.